src/lang/fr.json

```diff
@@ -0,0 +1,314 @@
+  "FSC.Configuration.Moon.YearResetX": "Réinitialiser l'année de référence tous les X ans",
+  "FSC.Configuration.Moon.YearX": "Réinitialiser les années lunaires de référence",
+  "FSC.Configuration.Moon.YearXHelp": "Réinitialiser l'année de référence des nouvelles lunes tous les x nombre d'années.",
+  "FSC.Configuration.Moon.Year": "Année de la Nouvelle Lune",
+  "FSC.Configuration.Moon.YearHelp": "L'année d'une nouvelle lune à utiliser comme référence.",
+  "FSC.Configuration.Moon.Month": "Mois de la Nouvelle Lune",
+  "FSC.Configuration.Moon.MonthHelp": "Le mois d'une nouvelle lune à utiliser comme référence.",
+  "FSC.Configuration.Moon.Day": "Jour de la Nouvelle Lune",
+  "FSC.Configuration.Moon.DayHelp": "Le jour d'une nouvelle lune à utiliser comme référence.",
+  "FSC.Configuration.Moons": "Lunes",
+  "FSC.Warn.Time.ActiveCombats": "Il y a un ou des combats actifs en cours, veuillez les résoudre avant de démarrer l'horloge en temps réel.",
+  "FSC.Warn.Notes.NotGM": "Il n'y a pas de MJ présent dans le jeu, un MJ doit être connecté pour que les joueurs puissent ajouter des notes.",
+  "FSC.Warn.Macros.GMUpdate": "Seul le MJ peut mettre à jour la date du calendrier!",
+  "FSC.Warn.Calendar.NotGM": "Il n'y a pas de MJ présent dans le jeu, un MJ doit être connecté pour que les joueurs puissent changer la date/l'heure.",
+  "FSC.Error.Note.NoTitle": "Impossible d'enregistrer une note sans titre !",
+  "FSC.Error.Note.RichText": "Veuillez enregistrer le contenu de l'éditeur enrichi.",
+  "FSC.Error.Note.NoSelectedDay": "Impossible d'ajouter une nouvelle note, il n'y a pas de jour sélectionné.",
+  "FSC.Error.Note.NoSelectedMonth": "Impossible d'ajouter une nouvelle note, il n'y a pas de mois sélectionné.",
+  "FSC.Error.Calendar.GMConfigure": "Vous devez être MJ pour configurer le calendrier.",
+  "FSC.Error.Calendar.GMCurrent": "Vous devez être MJ pour changer la date actuelle.",
+  "FSC.Module.Import": "Importer dans un Simple Calendar",
+  "FSC.Module.NoChanges": "Aucun changement",
+  "FSC.Module.CalendarWeather.Title": "Calendar/Weather détecté !",
+  "FSC.Module.CalendarWeather.Message": "Nous avons détecté que le module Calendar/Weather est activé pour ce monde.<br/><br/>Si vous souhaitez importer les paramètres de Calendar/Weather dans Simple Calendar, choisissez \"Importer dans le calendrier simple\", cela fera correspondre le Simple Calendar aux paramètres de Calendar/Weather.<br/><br/>Si vous souhaitez que Calendar/Weather corresponde au Simple Calendar, choisissez \"Exporter dans Calendar/Weather \", cela fera correspondre les paramètres Calendar/Weather à la configuration de Simple Calendars.<br/><br/>Sinon, vous pouvez choisir \"Aucun changement\" et les choses resteront telles quelles. Si vous souhaitez effectuer une importation /exporter plus tard, l'option sera disponible dans les paramètres généraux de Simple Calendar.<br/>",
```

Comment:
At one point we say "dans le calendrier simple", this should be changed to correctly reference the module name.